workload-team/app-infra-stamp.bicep

```diff
@@ -1030,21 +1036,6 @@ resource contosoPrivateDnsZone 'Microsoft.Network/privateDnsZones@2020-06-01' =
       registrationEnabled: false
     }
   }
-
-  // TODO: This is going to need to be solved. As this is configured below, it will not work in this topology.
-  // THIS IS BEING DONE FOR SIMPLICTY IN DEPLOYMENT, NOT AS GUIDANCE.
-  // Normally a workload team wouldn't have this permission, and a DINE policy
-  // would have taken care of this step.
-  resource linkToHub 'virtualNetworkLinks' = {
-    name: 'to_${hubVirtualNetwork.name}'
-    location: 'global'
-    properties: {
-      virtualNetwork: {
-        id: hubVirtualNetwork.id
```

Comment:
Shouldn't need the `existing` `hubVirtualNetwork` resource anymore at the top of the file.  Please remove.

Reply:
thanks for pointing this out @ckittel, 

done | addressed from 1d0042c8d2bd8cea8a6b45ce098bca51f12b50e8